src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.html

```diff
@@ -159,6 +161,16 @@
                       i18n>Cluster</span>
               </div>
             </div>
+
+            <div class="d-flex flex-column col-md-3">
```

Comment:
can you apply this same `col-md-3` to Cluster Status as well. col with some initial margin, `col-md-3 ms-4`